apps/site/next.config.js

```diff
@@ -1,4 +1,7 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+	reactStrictMode: true,
+	swcMinify: false,
```

Comment:
Question: any particular reason we need to disable the swc compiler?

Reply:
I noticed that we did this for the Hack at UCI 2023 website so I just copied this over.

Reply:
Looking at the docs, it's probably better if we enable it, especially since it's a lot faster than not using it. (https://nextjs.org/docs/architecture/nextjs-compiler)

Reply:
Yeah for the HackUCI site, it was to prevent an issue with minifying Cloudscape code. Shouldn't be necessary for new projects.